overlord/hookstate/ctlcmd/helpers.go

```diff
@@ -46,6 +46,10 @@ func getServiceInfos(st *state.State, snapName string, serviceNames []string) ([
 	if err != nil {
 		return nil, err
 	}
+	if len(serviceNames) == 0 {
```

Comment:
I wonder if we could make service name optional for snapctl start/stop/restart as well with this change in place. This would be a handy shortcut to say "start all my services". Anyway, that's potentially a metrial for another PR.

Reply:
I think it'd be good, but I'm biased.